sdk/spring/azure-spring-data-cosmosdb/QueryPartitionedCollection.md

```diff
@@ -0,0 +1,58 @@
+### How to Query Partitioned Azure Cosmos DB Collection
```

Comment:
What is the use of this file ?
This should be merged with README file, or move this topic to spring-data-cosmosdb developer's guide : https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/how-to-guides-spring-data-cosmosdb

Reply:
The content of this file has been included into README file about using partitioned query, thus I will remove this file.